ConsoleApp/Member.cs

```diff
@@ -0,0 +1,20 @@
+namespace BookLibrary
+{
+    public class Member
+    {
+        private static int _id;
```

Comment:
not critical, we have not even learnt it yet, but here you can use GUID to create unique IDs always, it's not the best way to generate ids